platform-demo-module/src/main/java/com/softicar/platform/demo/module/invoice/paid/DemoInvoicePaymentForm.java

```diff
@@ -0,0 +1,38 @@
+package com.softicar.platform.demo.module.invoice.paid;
+
+import com.softicar.platform.demo.module.DemoI18n;
+import com.softicar.platform.dom.elements.DomDiv;
+import com.softicar.platform.dom.elements.message.DomMessageDiv;
+import com.softicar.platform.dom.elements.message.style.DomMessageType;
+import com.softicar.platform.emf.form.EmfFormDelegator;
+import com.softicar.platform.emf.form.EmfFormMode;
+import com.softicar.platform.emf.form.IEmfFormFrame;
+
+public class DemoInvoicePaymentForm extends EmfFormDelegator<AGDemoInvoicePayment> {
+
+	private final DomDiv container;
+
+	public DemoInvoicePaymentForm(IEmfFormFrame<AGDemoInvoicePayment> frame, AGDemoInvoicePayment tableRow) {
+
+		super(frame, tableRow);
+		this.container = appendChild(new DomDiv());
+		form.setModeChangeCallback(this::changeMode);
+	}
+
+	private void changeMode(EmfFormMode mode) {
```

Comment:
changeMode should be called handleModeChange

Reply:
not possible, clashes with method name in IEmfForm